src/hackerspace_online/tests/test_management_commands.py

```diff
@@ -85,3 +89,25 @@ def test_initdb(self):
             self.assertTrue(Site.objects.exists())
 
             Tenant.objects.get(schema_name=apps.get_app_config('library').TENANT_NAME)  # no assert, but will throw exception if doesn't exist
+
+
+class GenerateContentTest(TenantTestCase, CommandMixin):
+    """ generate adds items to an existing tenant.
+    Dont need extensive testing as tests exist in "test_shell_utils.py"
+    """
+    name = "generate_content"
+
+    def test_added_rows_to_db(self):
+        """ test checks if "generate_content" adds objects to the db """
+        # cache initial numbers
+        quest_count = Quest.objects.count()
+        campaign_count = Category.objects.count()
+        user_count = User.objects.count()
+
+        # name is in a list?
+        self.call_command(self.tenant.schema_name, '--quiet')
+
+        # test if the command added new objects
+        self.assertNotEqual(Quest.objects.count(), quest_count)
```

Comment:
Can you make these counts precise?  Send some numbers to the generate_content command, then use those numbers here to make sure they match,

Reply:
done